### DISCUSSION TITLE
どの問題を解くか？

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#5] [General]
[NAFEMS](https://www.nafems.org/publications/resource_center/p18/)のどの問題を解くかご意見をお願いいたします。

**Comment by tkoyama010** [comment]
[Abaqusのベンチマーク](http://130.149.89.49:2080/v6.13/books/bmk/default.htm)は参考になりそうです。

**Reply by mmer547** [reply]
そうですね。そのベンチマークは参考になると思います。ただ、WebにあるAbaqusのベンチマークマニュアル自体を参照すると著作権的によくなさそうなので、各問題でベースとしている論文を参照する形で行きたいですね

**Comment by mmer547** [comment]
NAFEMSの問題は基本全部解いていこうと思っています。簡単な問題までで止まってるWebサイトも多いので、情報少なそうな問題から解いて行くのがいいかもしれないです

**Reply by tkoyama010** [reply]
了解です。まずは問題を追加する手順を整備したいので有名な [Cook's membrane](http://130.149.89.49:2080/v6.11/books/bmk/default.htm?startat=ch02s01ach137.html) はいかがでしょう？

**Reply by mmer547** [reply]
OSSのコードとかでもよく見るやつですね。それで行きましょう。

**Reply by tkoyama010** [reply]
[issueテンプレート](https://github.com/FEABeRjp/FEABeR/issues/new/choose)を作成しました。「問題の提案」を選択するとベンチマーク問題の提案ができます。ご意見をいただけるとありがたいです。

**Reply by mmer547** [reply]
後で試しに書いてみようと思いますが、コマーシャル資料の転載について注意書きがいるかどうかですね
